docs/modules/sort/dataflow_example.md

```diff
@@ -5,307 +5,1717 @@ sidebar_position: 3
+                            "inputField":{
+                                "type":"base",
+                                "name":"id",
+                                "formatInfo":{
+                                    "type":"long"
+                                }
+                            },
+                            "outputField":{
+                                "type":"base",
+                                "name":"id",
+                                "formatInfo":{
+                                    "type":"long"
+                                }
+                            }
+                        },
+                        {
+                            "type":"fieldRelationShip",
+                            "inputField":{
+                                "type":"base",
+                                "name":"name",
+                                "formatInfo":{
+                                    "type":"string"
+                                }
+                            },
+                            "outputField":{
+                                "type":"base",
+                                "name":"name",
+                                "formatInfo":{
+                                    "type":"string"
+                                }
+                            }
+                        },
+                        {
+                            "type":"fieldRelationShip",
+                            "inputField":{
+                                "type":"base",
+                                "name":"age",
+                                "formatInfo":{
+                                    "type":"int"
+                                }
+                            },
+                            "outputField":{
+                                "type":"base",
+                                "name":"age",
+                                "formatInfo":{
+                                    "type":"int"
+                                }
+                            }
+                        }
+                    ],
+                    "sinkParallelism":1,
+                    "catalogName":"hivecatlog",
+                    "database":"YOUR_DATABASE",
+                    "tableName":"YOUR_TABLE_NAME",
+                    "hiveConfDir":"YOUR_HIVE_CONF_DIR",
+                    "hiveVersion":"3.1.2",
+                    "hadoopConfDir":"YOUR_HADOOP_CONF_DIR"
+                }
+            ],
+            "relations":[
+                {
+                    "type":"baseRelation",
+                    "inputs":[
+                        "1"
+                    ],
+                    "outputs":[
+                        "2"
+                    ]
+                }
+            ]
+        }
+    ]
+}
+```
+
+## Transform examples
+
+Currently only supports string split, string regex replace, string regex replace first matched value, data distinct,data filter, regular join, and etc.
```

Comment:
add a link to support transformation:
https://github.com/yunqingmoswu/incubator-inlong-website/blob/INLONG-380/docs/modules/sort/overview.md#supported-transform

Reply:
It is a good idea